src/seedu/addressbook/commands/Command.java

```diff
@@ -17,6 +17,11 @@
     private int targetIndex = -1;
 
     /**
+     * returns true for command types that mutate the data
```

Comment:
It is better to mention it returns true if the Command *potentially* mutates the data (header comments specify a contract, so try to be as precise as possible). E.g. Adding a duplicate person will not mutate data but the corresponding AddCommand object will still return true for this method.